docs/configuration.md

```diff
@@ -197,23 +197,19 @@ This list also adds IP addresses which are trying to make DDoS attacks on the Pa
 
 The Splunk user has to add the API ID and API key to make HoneyDB API calls to retrieve the blocked IP list. Follow these steps to update the API ID and API key for the Cyences App:
 
-1. Navigate to the Cyences app and from the navigation bar go to **Settings > Configuration**.
+1. Navigate to the Cyences app and from the navigation bar go to **Cyences Settings > Cyences App Configuration > HoneyDB Configuration**.
 
-2. At the top of the dashboard there is a section for **HoneyDB Configuration**. 
-
-3. Update the **API ID** and **API Key** for the HoneyDB API. 
+2. Update the **API ID** and **API Key** for the HoneyDB API. 
 
 ![alt]({{ site.baseurl }}/assets/honeydb_config.png)
 
 ## **Configuration of Malicious IP Collector Server**
 
 The Splunk user has to add the API URL and Auth token to make API calls to Cyences Malicious IP List server to create malicious IP list and retrieve the latest malicious IP list from the server. Follow these steps to update the same for the Cyences App: 
 
-1. Navigate to the Cyences app and from the navigation bar go to **Settings > Configuration**.  
```

Comment:
Lets keep only two simple configuration section:
* HoneyDB configuration
* Malicious IP Configuration
(keep the current titles, titles here are for representation only)

remove extra details which are not important.